jwql/website/apps/jwql/templates/base.html

```diff
@@ -19,8 +19,8 @@
 		<!-- IMPORTANT: jQuery first, then popper, then Bootstrap -->
 		<script src="{{ static('js/jquery-3.3.1.min.js') }}?{% now 'U' %}"></script>
 		<script src="{{ static('js/tinysort.min.js') }}?{% now 'U' %}"></script>
-	    <script type="text/javascript" src="http://cdn.pydata.org/bokeh/release/bokeh-0.12.5.min.js"></script>
-	    <script src="http://cdn.pydata.org/bokeh/release/bokeh-widgets-0.12.5.min.js"></script>
+	    <script type="text/javascript" src="http://cdn.pydata.org/bokeh/release/bokeh-1.0.1.min.js"></script>
+	    <script src="http://cdn.pydata.org/bokeh/release/bokeh-widgets-1.0.1.min.js"></script>
```

Comment:
These should be 1.0.2 now, right? Will we need to update these each time we update Bokeh?

Reply:
Actually I think we should keep them at `1.0.1` until we update to `1.0.2` in our `environment.yml` file.  Ideally, we will update these as we update `bokeh` across our entire project, but unless we implement something that is super cutting-edge, I don't think things will break if we leave these alone for some time.

Reply:
I was using `1.0.2` in my attempts to debug the problems we were having, but i've switched it back to `1.0.1`